### PR TITLE
[SC-5695] Serialize error node display

### PR DESCRIPTION
**Pull request by NgoHarrison**

### CODE REVIEW

File: ee/vellum_ee/workflows/display/nodes/vellum/error_node.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            )
+            for variable_name, variable_id in self.node_input_ids_by_name.items()
+        ]
+
+        return {
+            "id": str(node_id),
+            "type": "ERROR",
+            "inputs": [node_input.dict() for node_input in node_inputs],
+            "data": {
+                "label": "Error Node",
```
Comment (noanflaherty) [inline ee/vellum_ee/workflows/display/nodes/vellum/error_node.py]: [blocking]

```suggestion
                "label": self.label,
```

File: ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_error_node_serialization.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+from vellum_ee.workflows.display.nodes.base_node_vellum_display import BaseNodeVellumDisplay
+from vellum_ee.workflows.display.workflows import VellumWorkflowDisplay
+from vellum_ee.workflows.display.workflows.get_vellum_workflow_display_class import get_workflow_display
+
+from tests.workflows.basic_error_node.workflow import BasicErrorNodeWorkflow
+
+
+def test_serialize_workflow():
+    # GIVEN a Workflow with a code execution node
```
Comment (noanflaherty) [inline ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_error_node_serialization.py]:
```suggestion
    # GIVEN a Workflow with an error node
```

**Review by noanflaherty — APPROVED** [review]

File: ee/vellum_ee/workflows/display/nodes/vellum/__init__.py [hunk 3]
```diff
@@ -31,4 +32,5 @@
     "BasePromptDeploymentNodeDisplay",
     "BaseFinalOutputNodeDisplay",
     "BaseTryNodeDisplay",
+    "BaseErrorNodeDisplay"
```
Comment (noanflaherty) [inline ee/vellum_ee/workflows/display/nodes/vellum/__init__.py]: nit: Can we make these in alphabetical order?

**Review by kirawoods — COMMENTED** [review]
✅ once Noa's comments are addressed

**Pull request merged**